src/app/clusters/application-basic-server/application-basic-server.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *
+ *    Copyright (c) 2021 Silicon Labs
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+/****************************************************************************
+ * @file
+ * @brief Routines for the Application Launcher plugin, the
+ *server implementation of the Application Launcher cluster.
+ *******************************************************************************
+ ******************************************************************************/
+
+#include <app/Command.h>
+#include <app/common/gen/enums.h>
+#include <app/util/af.h>
+#include <string>
+
+bool applicationBasicClusterChangeApplicationStatus(EmberAfApplicationBasicStatus status, chip::EndpointId endpoint);
```

Comment:
Again, should be using the callback zapt bits, as a followup.

Reply:
Created an [issue](https://github.com/project-chip/connectedhomeip/pull/7906#discussion_r662765736) for follow up.